components/Ink/src/MDCInkView.h

```diff
@@ -54,27 +54,36 @@ typedef NS_ENUM(NSInteger, MDCInkStyle) {
 @property(nonatomic, assign) MDCInkStyle inkStyle;
 
 /** The foreground color of the ink. The default value is defaultInkColor. */
-@property(nonatomic, strong, null_resettable) UIColor *inkColor;
+@property(nonatomic, strong, nonnull) UIColor *inkColor;
 
 /** Default color used for ink if no color is specified. */
 @property(nonatomic, strong, readonly, nonnull) UIColor *defaultInkColor;
 
 /**
  Maximum radius of the ink. If the radius <= 0 then half the length of the diagonal of self.bounds
  is used. This value is ignored if @c inkStyle is set to |MDCInkStyleBounded|.
+
+ Ignored if updated ink is used.
  */
 @property(nonatomic, assign) CGFloat maxRippleRadius;
 
+/**
+ Use the older legacy version of the ink ripple. Default is YES.
```

Comment:
This will eventually be changed to NO, but for the time being we need to keep existing clients unchanged. I will outline the migration plan in a design doc.